lib/matplotlib/tests/test_mathtext.py

```diff
@@ -12,7 +12,7 @@
 from matplotlib import mathtext
 
 math_tests = [
-    r'$a+b+\dots+\dot{s}+\ldots$',
+    r'$a+b+\dot s+\dot{s}+\ldots$',
```

Comment:
According to the TeX definitions,  ``\dots`` should not produce three dots: only ``\ldots`` should. ``\dots`` would give an error.